beacon-chain/sync/validate_blob.go

```diff
@@ -159,35 +164,39 @@ func (s *Service) validateBlobPostSeenParent(ctx context.Context, sBlob *eth.Sig
 	if err != nil {
 		return pubsub.ValidationIgnore, err
 	}
-	if err := verifyBlobSignature(parentState, sBlob); err != nil {
-		log.WithError(err).WithFields(blobFields(blob)).Debug("Failed to verify blob signature")
-		return pubsub.ValidationReject, err
-	}
+	// TODO: replace this with verification routine
```

Comment:
Why not call here the proof verification helper?

Reply:
We'll do that in https://github.com/prysmaticlabs/prysm/pull/13197